rover_msgs/include/rovus_lib/camera_info.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef __CAMERA__INFO__HPP__
+#define __CAMERA__INFO__HPP__
+
+#include <map>
+#include <string>
+
+namespace CameraInfo
+{
+    const std::map<std::string, std::string> CameraName = {{"rtsp://rover:roverrover@192.168.144.30:554/1/h264major", "Main"},
+                                                           {"rtsp://rover:roverrover@192.168.144.31:554/1/h264major", "Antenne"},
+                                                           {"rtsp://rover:roverrover@192.168.144.31:554/1/h264major", "TestCam"}};
+}
+
+#endif
```

Comment:
Sugg: After #endif add comments to the initial conditions, it's clearer when there's multiple #if / #endif in the file
```suggestion
#endif // __CAMERA__INFO__HPP__
```